compiler/test/dotty/tools/dotc/reporting/CodeActionTest.scala

```diff
@@ -179,6 +179,79 @@ class CodeActionTest extends DottyTest:
       ctxx = ctxx
       )
 
+  @Test def addNN =
```

Comment:
Let's also test function applications where the argument need `.nn` and some complex expressions, like a match or if.